### ISSUE TITLE
Possible to make process name easily changeable?

### ISSUE THREAD

**Issue by NonQuixote** [UltimateFishbot/UltimateFishbot#74]
Warden has the ability to scan active process names, although whether or not or how often it does is a point of contention in any forum.  Most of the bots that I've used have an easy way to change the process name to something innocuous, like "sillyname.exe".  Would it be possible to easily add a feature like this to UFB?  I ask because while I recently began using the fishbot I am clueless when it comes to writing or compiling code.  If it's a PITA a newb friendly post on what to change and how to recompile with the change would also be greatly appreciated!

If not, thanks for taking a look, and thanks to all who have contributed to keeping this bot going!

**Comment by gmillerd** [comment]
It should be a priority to alter this.

On Tue, Feb 2, 2016 at 8:37 AM, NonQuixote notifications@github.com wrote:

> Warden has the ability to scan active process names, although whether or
> not or how often it does is a point of contention in any forum Most of the
> bots that I've used have an easy way to change the process name to
> something innocuous, like "sillynameexe" Would it be possible to easily add
> a feature like this to UFB? I ask because while I recently began using the
> fishbot I am clueless when it comes to writing or compiling code If it's a
> PITA a newb friendly post on what to change and how to recompile with the
> change would also be greatly appreciated!
> 
> If not, thanks for taking a look, and thanks to all who have contributed
> to keeping this bot going!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/UltimateFishbot/UltimateFishbot/issues/74.

## 

Gabriel Millerd

**Comment by Aquenon** [comment]
I don't use bots normally. The only time I think about is for fishing. I am not worried about players seeing me, because the only things I have left to do can be done in my garrison or the shipyard, both of them instances areas. 

But, I am weary now that I know Warden can look at running processes. I am new to compiling as all the programming I have done is run through an interpreter instead (VBA in Office, some JavaScript), no matter how simple or complex I make them. But I have looked up how to do it on the command line.

My question is, what can I change in the source code in order to give the compiled executable a different name so as not to be detected?

Thanks

**Comment by Gasur** [comment]
To change the process name, you simply just need to change the name of the .exe file. If you wish to remove, or change the process description you need to edit AssemblyInfo.cs under Properties.

**Comment by KrazyTurk** [comment]
Maybe someone that knows better than I can chime in, but I thought Warden explicitly wasn't allowed to scan running processes as this would be considered and invasion of privacy?

**Comment by NonQuixote** [comment]
After deciding not to use this because the process name couldn't easily be changed by a non-coder, I just fished manually.  Changing the name of the .exe file did nothing for me btw, UltimateFishBot (32bit) is still the process name that shows up in task manager, even when the executable file name is changed to something silly like "jimjam".  I installed Visual Studio, but I can't see spending days learning to use it just to automate fishing.

So, thanks for the effort!  It seems to work fine for a lot of folks, I'm just a bit gunshy after having been banned a few times...lol.
